packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -156,13 +161,9 @@ contract DeployImplementationsInput is BaseDeployIO {
         return _protocolVersionsProxy;
     }
 
-    function superchainProxyAdmin() public returns (ProxyAdmin) {
-        SuperchainConfig proxy = this.superchainConfigProxy();
-        // Can infer the superchainProxyAdmin from the superchainConfigProxy.
-        vm.prank(address(0));
-        ProxyAdmin proxyAdmin = ProxyAdmin(Proxy(payable(address(proxy))).admin());
-        require(address(proxyAdmin) != address(0), "DeployImplementationsInput: not set");
-        return proxyAdmin;
```

Comment:
Why is this no longer needed?

Reply:
@mds1 asked me to let the user specify any address as the OPCM proxy owner. Since the owner was previously being set to the superchain proxy admin, we no longer need this method as the OPCM proxy owner is passed in in the DeployInput.